pkgs/by-name/ta/tangle/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  fetchurl,
+  # Build depends
+  cmake,
+  qt6,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tanglet";
+  version = "1.6.5";
+
+  src = fetchFromGitHub {
+    owner = "gottcode";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-xhFxH6yBKZwI5+5BX5F4H325h2V7hOLe7UocUMcrQAo=";
+  };
+
+  buildInputs = [
+    cmake
+    qt6.full
```

`qt6.full` gets too many packages that are not needed.
```suggestion
    qt6.qtbase
    qt6.qtdeclarative
    qt6.qttools
    qt6.wrapQtAppsHook
```